app-metadata.html

```diff
@@ -89,6 +89,16 @@
             // title is a special case
             if (name === 'title') {
               document.title = this.data[name];
+            } else if (name.substring(0, 3) === 'og:') {
```

Comment:
Instead of putting the test here, which duplicates more code in the block, it might be slightly neater to just add the test for the name / property setting (the only part that differs) so there are fewer code paths.

i.e.

```
} else {
  // re-use any already-created meta-tags if possible
  if (this._meta.hasOwnProperty(name)) {
    this._meta[name].content = this.data[name];
  } else {
    var m = document.createElement('meta');
    if (name.substring(0, 3) === 'og:') {
      m.setAttribute('property', name);
    } else {
      m.name = name;
    }
    m.content = this.data[name];
    document.head.appendChild(m);
    this._meta[name] = m;
  }
}
```